ember/package.json

```diff
@@ -12,75 +12,103 @@
   },
   "scripts": {
     "build": "ember build --environment=production",
-    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*",
+    "lint": "npm-run-all --aggregate-output --continue-on-error --parallel \"lint:!(fix)\"",
+    "lint:fix": "npm-run-all --aggregate-output --continue-on-error --parallel lint:*:fix",
     "lint:hbs": "ember-template-lint .",
-    "lint:js": "eslint .",
+    "lint:hbs:fix": "ember-template-lint . --fix",
+    "lint:js": "eslint . --cache",
+    "lint:js:fix": "eslint . --fix",
     "start": "ember serve",
     "start-proxy": "ember server --proxy http://localhost:8000/",
     "test": "npm-run-all lint:* test:*",
     "test:ember": "ember test"
   },
   "devDependencies": {
-    "@adfinis-sygroup/eslint-config": "^1.4.2",
-    "@ember/optional-features": "^1.3.0",
+    "@adfinis-sygroup/eslint-config": "^1.5.0",
+    "@apollo/client": "^3.4.16",
+    "@ember/optional-features": "^2.0.0",
+    "@ember/render-modifiers": "^2.0.0",
+    "@ember/test-helpers": "^2.6.0",
     "@glimmer/component": "^1.0.4",
     "@glimmer/tracking": "^1.0.4",
+    "@projectcaluma/ember-core": "^11.0.0-beta.4",
+    "@projectcaluma/ember-form": "^11.0.0-beta.12",
+    "@projectcaluma/ember-form-builder": "^11.0.0-beta.8",
+    "@projectcaluma/ember-testing": "^11.0.0-beta.3",
+    "@projectcaluma/ember-workflow": "^11.0.0-beta.4",
```

Comment:
Do you need all of those? I'm pretty sure you don't need testing - you should then also remove mirage

Reply:
Removed